src/utils/presets.ts

```diff
@@ -213,13 +213,30 @@ export const uiliciousPreset = {
+  width: 400,
+  height: 400,
+  margin: 0,
+  dotsOptions: { color: '#252f3f', type: 'classy-rounded' },
+  cornersSquareOptions: { color: '#252f3f', type: 'dot' },
+  cornersDotOptions: { color: '#f05252', type: 'dot' },
+  imageOptions: { margin: 4 },
+  style: { borderRadius: '22pxpx', background: '#ffffff' }
```

Comment:
```suggestion
  style: { borderRadius: '22px', background: '#ffffff' }
```

Reply:
I grabbed the configuration directly from the exported JSON file when I exported my preset.
So there might be a similar issue in the exporting function. We may need to take a look into that. 😅

Reply:
oh i see, thanks for letting me know, i'll check on that!

Reply:
i just tested on your branch, the exported config doesn't have the typo, so i think there's no issue for now 😊 but let me know if it happens again by opening an issue.